velox/expression/tests/ArgumentTypeFuzzer.cpp

```diff
@@ -72,10 +48,14 @@ void ArgumentTypeFuzzer::determineUnboundedTypeVariables() {
     // Random randomType() never generates unknown here.
     // TODO: we should extend randomType types and exclude unknown based
     // on variableInfo.
-    bindings_[variableName] = randomType(rng_);
+    bindings_[variableName] = randType();
   }
 }
 
+TypePtr ArgumentTypeFuzzer::randType() {
+  return velox::randType(rng_, 2);
```

Comment:
wondering if we would get any value from increasing the depth here?

Reply:
I started with maxDepth = 5, but Fuzzer was running very slowly. Hence, reduced to 2.